lib/travis/api/v3/models/job.rb

```diff
@@ -10,5 +12,11 @@ class Models::Job < Model
     belongs_to :owner, polymorphic: true
     serialize :config
     serialize :debug_options
+
+    if Travis::Config.logs_api_enabled?
+      def log
```

Comment:
I think we can simplify some later code (anywhere you call job.log or RemoteLog conditionally) if you do this:
``` ruby 
def log
  return super unless Travis::Config.logs_api_enabled?
  @log ||= RemoteLog.find_by_job_id(id)
end
```